index.css

```
@@ -0,0 +1,171 @@
+html body {
+    font-family: Helvetica, sans-serif;
+    font-size: 22px;
+    margin: 0;
+    padding: 0;
+    background-color: black;
```

Its bad practice to use built in CSS colors. Use the hex if possible.
```suggestion
    background-color: #000000;
```